config/environments/production.rb

```diff
@@ -60,8 +60,6 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  config.action_controller.asset_host = config.action_mailer.asset_host  = 'https://libraries.io'
```

Comment:
Why losing this?

Reply:
Its redundant and I was confused by it at first. I believe the asset host is for if we were uploading to a CDN or something as a part of our build process. Removing this line serves them from the exact same location but makes it easier to test production assets locally as well.

Reply:
I don't understand what you mean by "testing production assets locally" -- for non-production (testing), we should be using the non-production environment

Reply:
Ah, all I meant was that if it is set to libraries.io I can't confirm locally that my changes to these configs actually work. By deleting that, all it does is use the relative path instead of prefixing a host which should end up with the same address.